main.py

```diff
@@ -1,7 +1,21 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
-
+from db.models import Actor, Genre
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. However, this is corrected later in the code, so it might be intentional for demonstration purposes.